l10n_ch_payment_slip/payment_slip.py

```diff
@@ -556,6 +564,8 @@ def _draw_bank(self, canvas, print_settings, initial_position, font, bank):
             if not line:
                 continue
             text.textLine(line)
+        line = (bank.zip + '' or '') + ' ' + (bank.city or '')
```

Comment:
I guess you do `+ ''` to cast to string.

Prefer to use 
```
"%s %s" % (bank.zip or '', bank.city or '')
```
It's usually preferable instead of doing multiple concatenations: http://programmers.stackexchange.com/questions/304445/why-is-s-better-than-for-concatenation

However, in your case you might want to get rid of the extra space if one or the other is not defined.

Thus you can use the `join` method on a list of string:

```
line = [str(bank.zip or ''), bank.city]
line = ' '.join([s for s in line if s])
```